lib/rules/validate-jsdoc/check-param-existence.js

```diff
@@ -0,0 +1,29 @@
+module.exports = validateCheckParamExistence;
+module.exports.scopes = ['function'];
+module.exports.options = {
+    checkParamExistence: {allowedValues: [true]}
+};
+
+/**
+ * validator for check-param-existence
+ * @param {(FunctionDeclaration|FunctionExpression)} node
+ * @param {Function} err
+ */
+function validateCheckParamExistence(node, err) {
+    if (!node.jsdoc) {
```

Comment:
This could be another check like `!node.jsdoc.valid`. But I'm not sure we really need this check here since this rule for `function` scope ;-)